spikes/2023-12-20-NotOctreeSplitting.md

```diff
@@ -0,0 +1,57 @@
+# Not Octree Splitting
+
+## [Reference branch](https://github.com/equinor/rvmsharp/tree/Spike/NotOctree)
+
+## Goal
+
+By octree splitting one consequence is that at the center of the octree, for instance, there can be lot of sectors with aligned edges. This means that if the user is moving, he would suddenly move from one set of sectors into a whole new set of sectors. By not splitting in an octree fashion, but rather make depths of not aligned sectors, the aligment problem could be avoided. The positives about oct tree splitting would hopefully not be lost.
```

Comment:
there can be lot of sectors with aligned edges/faces.

Reply:
This means that if a user has rapid transitions between sectors with aligned edges/faces, a large set of sectors will load for each transition. This could decrease user experience due to slow loading of geometry and lag.